src/options/popup.js

```diff
@@ -1,6 +1,26 @@
 "use strict"
 
-import { statusDeny, statusAllow, statusDismiss, categories }  from "../constants/index.js"
+import {
+  STATUS_DENY, STATUS_ALLOW, STATUS_DISMISS, CATEGORIES,
```

Comment:
Please just list them one below each other, and just sort alphabetically :)

Reply:
formatting issues will be foxed when (if) we add later on pretties and eslint